code/main/edge/buildEdgeRequestURL.brs

```diff
@@ -13,10 +13,11 @@
 
 ' ***************************** MODULE: buildEdgeRequestURL *******************************
 
-function _adb_buildEdgeRequestURL(configId as string, requestId as string, path as string, edgeDomain = invalid as dynamic) as string
+function _adb_buildEdgeRequestURL(configId as string, requestId as string, path as string, locationHint as dynamic, edgeDomain = invalid as dynamic) as string
```

Comment:
[Nit] Could you add documentation comments on this function to show the correct formation of the 
`path` parameter?

Reply:
updated